test/e2e/common/setup-kubestellar-ocp.sh

```diff
@@ -51,27 +51,32 @@ kflex create its1 --type vcluster -p ocm $disable_chatty_status
 
 :
 : -------------------------------------------------------------------------
-: Install singleton status return addon in IMBS1
+: Install singleton status return addon in ITS1
 :
 wait-for-cmd kubectl --context its1 api-resources "|" grep managedclusteraddons
 helm --kube-context its1 upgrade --install status-addon -n open-cluster-management oci://ghcr.io/kubestellar/ocm-status-addon-chart --version v${OCM_STATUS_ADDON_VERSION}
 :
 : -------------------------------------------------------------------------
 : Create a Workload Description Space wds1 directly in KubeFlex.
 :
-kflex create wds1 -p kubestellar
+kflex create wds1 -p kubestellar $disable_chatty_status
+kubectl --context kscore label cp wds1 kflex.kubestellar.io/cptype=wds
+
 echo "wds1 created."
 
 :
 : -------------------------------------------------------------------------
 : Run OCM transport controller in a pod
 :
-kflex ctx
-bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/deploy-transport-controller.sh) wds1 its1
-wait-for-cmd '(kubectl -n wds1-system wait --for=condition=Ready pod/$(kubectl -n wds1-system get pods -l name=transport-controller -o jsonpath='{.items[0].metadata.name}'))'
+helm --kube-context kscore upgrade --install ocm-transport-plugin oci://ghcr.io/kubestellar/ocm-transport-plugin/chart/ocm-transport-plugin --version ${OCM_TRANSPORT_PLUGIN} \
+ --set transport_cp_name=its1 \
+ --set wds_cp_name=wds1 \
+ -n wds1-system
 
-echo "transport controller is running."
+wait-for-cmd '(kubectl --context kscore -n wds1-system wait --for=condition=Ready pod/$(kubectl --context kscore -n wds1-system get pods -l name=transport-controller -o jsonpath='{.items[0].metadata.name}'))'
```

Comment:
This works, but is misleading and more complex than it needs to be. Note that bash does not require those curly and square braces to be quoted:

```
bash-3.2$ kubectl get pods -n wds1-system kubestellar-controller-manager-78fc476cbf-mzkn5 -o jsonpath={.status.conditions[0].type}
Initializedbash-3.2$ 
```

Note that this line has four squotes, and squotes do not nest; squoted input is not parsed, not even for `$(...)`. The way bash parses this command line is as follows.

1. word `wait-for-cmd`
2. a word composed by concatenating three things: (a) some squoted content, (b) `{.items[0].metadata.name}`, and (c) squoted double close paren.

Reply:
And this construct predates this PR. Does not have to be addressed in this PR.

Reply:
Agreed!